python/pyspark/sql/functions.py

```diff
@@ -92,14 +98,16 @@ def _():
 _functions_1_4 = {
     # unary math functions
     'acos': 'Computes the cosine inverse of the given value; the returned angle is in the range' +
-            '0.0 through pi.',
-    'asin': 'Computes the sine inverse of the given value; the returned angle is in the range' +
-            '-pi/2 through pi/2.',
-    'atan': 'Computes the tangent inverse of the given value.',
+            '0.0 through pi.\n\n:param col: float column, units in radians.',
+    'asin': """Computes the sine inverse of the given value; the returned angle is in the range
+            -pi/2 through pi/2.
+
+            :param col: float column, units in radians.""",
```

Comment:
I was trying two different ways to see which worked best. I now use the `:param col:` version